src/identity/apis/account.ts

```diff
@@ -0,0 +1,95 @@
+// APIs
+import {
+  getAccount,
+  getAccounts,
+  patchAccount,
+  putAccountsDefault,
+  Account,
+  IdentityAccount,
+  UserAccount,
+} from 'src/client/unityRoutes'
+
+// Types
+import {
+  GenericError,
+  ServerError,
+  UnauthorizedError,
+  UnprocessableEntityError,
+} from 'src/types/error'
+
+export interface CurrentAccount extends IdentityAccount {
+  billingProvider?: 'zuora' | 'aws' | 'gcm' | 'azure'
+}
+
+// get a list of the user's accounts
+export const getUserAccounts = async (): Promise<UserAccount[]> => {
+  const response = await getAccounts({})
+
+  if (response.status === 401) {
+    throw new UnauthorizedError(response.data.message)
+  }
+  if (response.status === 500) {
+    throw new ServerError(response.data.message)
+  }
+  if (!Array.isArray(response.data)) {
+    throw new GenericError('No account found')
+  }
+
+  return response.data
+}
+
+// fetch more details about one of the user's accounts
+export const fetchAccountDetails = async (
+  accountId: string | number
+): Promise<Account> => {
+  const accountIdString = accountId.toString()
+
+  const response = await getAccount({
+    accountId: accountIdString,
+  })
+
+  if (response.status === 401) {
+    throw new UnauthorizedError(response.data.message)
+  }
+
+  if (response.status === 500) {
+    throw new ServerError(response.data.message)
+  }
+
+  const accountDetails = response.data
```

Comment:
`nit`: feel free to ignore but I think this is kind of unnecessary.

Reply:
If we edit further, I'll implement. 👍  This is part of the existing account API, which I didn't make changes to.